src/response/generate_api_token_response.rs

```diff
@@ -0,0 +1,17 @@
+use std::time::SystemTime;
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ApiKeyWithEndpoint {
+    pub api_key: String,
```

Comment:
What's the difference between an api key and an api token?

Reply:
We had this discussion in the design meeting. Basically we are calling the base64 encoded `(endpoint, api_key)` the `api_token`. We thought that it made the most sense because that string they will just treat as a token, and there is no need for them to base64 decode that string. A user should only have to deal with `_tokens`. So `refresh_token`, `api_token`, `session_token`.

Reply:
This also matches what is in the protos

Reply:
Should this struct be called ApiToken with a `/// Comment` to that effect?

Reply:
yea I can change the name and add a comment

Reply:
added comment in this commit https://github.com/momentohq/client-sdk-rust/pull/132/commits/a10baf2df4d874788a9eae80e49a54fbedf939bd